spec/std/ini_spec.cr

```diff
@@ -1,6 +1,53 @@
 require "spec"
 require "ini"
 
+class INI
```

Comment:
Why is this in the spec??

Reply:
Sorry, I've pasted this by mistake. I revert this ASAP